### PR TITLE
bugfix/20264-toogling-inverted-packedbubble

### LINKED ISSUE #20264: Toggle Chart Type Not Rendering Correctly In All Cases
#### Expected behaviour
When changing chart type from packedbubble to bar then back to packedbubble the chart should look as it did on initial load of the packedbubble.

#### Actual behaviour
When you change back to initial chart type the chart group width appears to be clipped so the packedbubble is cut off.

#### Live demo with steps to reproduce
https://jsfiddle.net/8sLhjptq/3/

Use export menu and click on "Switch chart", This changes the chart from initial packedbubble to bar.
Click on "Switch chart" again and this changes the chart back to packedbubble. Note that chart is clipped on the right.

#### Product version
HighCharts 11.2

Worked under HighCharts v9

#### Affected browser(s)
All

### PR DESCRIPTION
**Pull request by hubertkozik**
Fixed #20264, the packed bubble series was not rendered correctly in an inverted chart.

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206127404572389

### CODE REVIEW

**Review by KacperMadej — APPROVED** [review]
👍

File: ts/Core/Series/Series.ts [hunk 1]
```diff
@@ -2394,6 +2394,13 @@ class Series {
             seriesBox.height = yAxis.len;
         }
 
+        // If chart is set to inverted and there are no axes, clip box shouldn't
+        // be inverted (#20264)
+        if (chart.inverted && !xAxis && !yAxis) {
+            [seriesBox.width, seriesBox.height] =
+                [seriesBox.height, seriesBox.width];
+        }
+
```
[the remark this answers is not in the document]
Comment (TorsteinHonsi) [reply ts/Core/Series/Series.ts:2403]: PS: The `chart.clipBox` seems to suffer from the same issue here, and continues to do so with the current state of this PR: https://jsfiddle.net/highcharts/oydrk26s/

[the remark this answers is not in the document]
Comment (hubertkozik) [reply ts/Core/Series/Series.ts:2403]: Yes, I added this code to fix the issue from the demo that you attached: https://jsfiddle.net/BlackLabel/7qLo2x46/ and added a unit test specially for that. 

I have tested it on this demo (`packedbubble` without axes): https://jsfiddle.net/BlackLabel/uv1943nc/

I am not sure about firing something like this after `propFromSeries`, I tried, but always gave me bad results, that's why I decided to change only `series.clipBox` to not ruin other logic. Besides that, currently, it works like this for any series with axes, `chart.clipBox` is calculated and then `series.clipBox` is overwritten in `series.getClipBox` with `axis.len` value.

[the remark this answers is not in the document]
Comment (TorsteinHonsi) [reply ts/Core/Series/Series.ts:2403]: Okay, I understand now. The `chart.clipBox` is actually correct after the update (it should be inverted). What you made sure is that the series that do not support inversion (packedbubble, pie etc), are not clipped to an inverted clip box. I'll do some more checking then come back. For example, some series like sankey or tree graphs support inversion but have no axes. Maybe we need a series flag like `invertible` if we can't find some existing property to check for.

**Review by TorsteinHonsi — APPROVED** [review]

**Pull request merged**